grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy

```diff
@@ -230,6 +231,7 @@ class StockMovementService {
     void updateRequisitionStatus(String id, RequisitionStatus status) {
 
         log.info "Update status ${id} " + status
+        // TODO: In Grails the get below should be replaced by the data service get that joins the Events
```

Comment:
here it sohuld be replaced by the data service query, something like `getWithEvents` that would `@Join('events')`
I know that @drodzewicz has run into the same issue but with approvers, so most probably we'll have to write the query using `@Query` and join those two tables (if I forget - remember to take care of **left** join there)

Reply:
You *could* (but don't have to) add a getWithEvents() to the Requisition class now.

Reply:
yeah, but since those data service annotations are not available in Grails 1, I would anyway have to write this HQL/SQL query by myself using `.executeQuery`, wouldn't I?